### PR TITLE
Create s3 accounts script

### PR DESCRIPTION
**Pull request by soniyamoholkar**
# Backend

## Problem Statement
<pre>
Story Ref (if any):
</pre>
## Unit testing on RPM done
<pre>
No
</pre>
## Problem Description
<pre>
Script required to create s3 acounts, IAM users, buckets
</pre>
## Solution
<pre>
Python script created
</pre>
## Unit Test Cases 
<pre>
Tested on local VM
</pre>
Signed-off-by: Soniya Moholkar

### CODE REVIEW

File: csm/conf/create_accounts.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import time
+import io
+import argparse
+import traceback
+import sys
+import os
+import pathlib
+
+
+class CreateAccount:
```
Comment (ajaysrivas) [inline csm/conf/create_accounts.py:12]: Use a noun for class name.

File: csm/conf/create_accounts.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import requests
```
Comment (ajaysrivas) [inline csm/conf/create_accounts.py:1]: Move this file to test folder.

**Review by ajaysrivas — CHANGES_REQUESTED** [review]

File: csm/conf/create_accounts.py (hunk 1 above)
Comment (soniyamoholkar) [reply csm/conf/create_accounts.py]: Done.

File: csm/conf/create_accounts.py (hunk 2 above)
Comment (soniyamoholkar) [reply csm/conf/create_accounts.py]: moved file to /test/s3

**Comment by kupranay** [comment]
We dont require this code in cortx 1.0 branch hence closing it. @soniyamoholkar  if not already raised please raise it for main with above comments fixed

**Pull request closed without merging**